<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtmSplashScreen</class>
 <widget class="QMainWindow" name="AtmSplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <widget class="QLabel" name="version_label">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>17</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border: 0px solid black;
}
QLabel{
color: rgb(32, 74, 135);
}</string>
    </property>
    <property name="text">
     <string>v.1.0.1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>580</width>
      <height>380</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;



</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="atm_company_name_label">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>40</y>
       <width>510</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS PGothic</family>
       <pointsize>25</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border: 0px solid rgb(32, 74, 135);
}
QLabel{
color: rgb(32, 74, 135);
}</string>
     </property>
     <property name="text">
      <string>Company name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="atm_label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>300</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS PGothic</family>
       <pointsize>90</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
background: transparent;
border: 0px solid black;
}
QLabel{
color: rgb(32, 74, 135);
}</string>
     </property>
     <property name="text">
      <string>ATM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="text_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>280</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS PGothic</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
border: 0px solid  rgb(32, 74, 135);
}
QLabel{
color: rgb(32, 74, 135);
}</string>
     </property>
     <property name="text">
      <string>press &lt;Enter&gt; to start</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
